// auth-identifier/service.js
function _0x471c(_0x212737, _0x295a91) {
  const _0x338f61 = _0x5601();
  return (
    (_0x471c = function (_0x365930, _0x1aebec) {
      _0x365930 = _0x365930 - (0x21 * -0x21 + -0x2d2 + 0x879 * 0x1);
      let _0x74173b = _0x338f61[_0x365930];
      return _0x74173b;
    }),
    _0x471c(_0x212737, _0x295a91)
  );
}
function _0x5601() {
  const _0xb6b58e = [
    "Server\x20Err",
    "get",
    "rWqwA",
    "/setting.m",
    "3308465rfrILv",
    "No\x20data\x20fo",
    "Success!!",
    "und!",
    "./auth",
    "json",
    "findOne",
    "1207218hKmjOk",
    "eLlJH",
    "express",
    "exports",
    "1116662oTVRbS",
    "message",
    "387712eKBjEi",
    "3IRxtdt",
    "status",
    "15460479IIAwuQ",
    "auth",
    "1585696HqScQd",
    "Router",
    "583982KhUETh",
    "odel",
    "er/setting",
    "8FOdtXt",
    "../../serv",
    "qcnEO",
    "log",
  ];
  _0x5601 = function () {
    return _0xb6b58e;
  };
  return _0x5601();
}
const _0xe1f033 = _0x471c;
(function (_0x1687a6, _0x54a5b2) {
  const _0x56e302 = _0x471c,
    _0x562f35 = _0x1687a6();
  while (!![]) {
    try {
      const _0x5c13a0 =
        parseInt(_0x56e302(0x183)) / (0xe0 * 0x3 + -0x2480 + -0xb1 * -0x31) +
        -parseInt(_0x56e302(0x181)) / (-0x2f0 * -0x5 + 0x20f6 + 0x1 * -0x2fa4) +
        (-parseInt(_0x56e302(0x184)) /
          (-0x21 * -0x97 + 0x1eb6 + -0x322a * 0x1)) *
          (parseInt(_0x56e302(0x169)) / (0x1577 + -0x18c7 + 0x354)) +
        -parseInt(_0x56e302(0x176)) / (-0xf50 + 0x1d * 0xd + -0x2 * -0x6ee) +
        -parseInt(_0x56e302(0x17d)) / (0x349 * 0x6 + 0x269e + -0x3a4e) +
        parseInt(_0x56e302(0x16b)) / (-0xd25 + -0x3d3 * -0x5 + -0x5f3) +
        (-parseInt(_0x56e302(0x16e)) /
          (-0x3 * -0x68c + -0x82c * -0x1 + 0x4 * -0x6f2)) *
          (-parseInt(_0x56e302(0x167)) / (0x24d * -0xf + 0x1843 + 0xa49));
      if (_0x5c13a0 === _0x54a5b2) break;
      else _0x562f35["push"](_0x562f35["shift"]());
    } catch (_0x41f4b7) {
      _0x562f35["push"](_0x562f35["shift"]());
    }
  }
})(_0x5601, -0x6005e + -0x9018c + 0x14ac60);
const express = require(_0xe1f033(0x17f)),
  router = express[_0xe1f033(0x16a)](),
  data = require(_0xe1f033(0x17a)),
  Setting = require(_0xe1f033(0x16f) +
    _0xe1f033(0x16d) +
    _0xe1f033(0x175) +
    _0xe1f033(0x16c));
router[_0xe1f033(0x173)]("/", async (_0x5cf95f, _0x404886) => {
  const _0x3a9e5c = _0xe1f033,
    _0x2207c6 = {
      eLlJH: _0x3a9e5c(0x177) + _0x3a9e5c(0x179),
      rWqwA: _0x3a9e5c(0x178),
      qcnEO: _0x3a9e5c(0x172) + "or",
    };
  try {
    const _0x48a4b5 = await Setting[_0x3a9e5c(0x17c)]({});
    if (!_0x48a4b5 || !data[_0x3a9e5c(0x168)])
      return _0x404886[_0x3a9e5c(0x166)](-0x53d + -0x4 * -0x4d5 + 0x1 * -0xd4f)[
        _0x3a9e5c(0x17b)
      ]({ status: ![], message: _0x2207c6[_0x3a9e5c(0x17e)] });
    return _0x404886[_0x3a9e5c(0x166)](
      0x5 * -0x44e + 0x1 * 0x6af + -0x81 * -0x1f
    )[_0x3a9e5c(0x17b)]({
      status: !![],
      message: _0x2207c6[_0x3a9e5c(0x174)],
      setting: _0x48a4b5,
    });
  } catch (_0x3af8c7) {
    return (
      console[_0x3a9e5c(0x171)](_0x3af8c7),
      _0x404886[_0x3a9e5c(0x166)](0x1c1 * 0x13 + 0x504 + -0x1 * 0x2463)[
        _0x3a9e5c(0x17b)
      ]({
        status: ![],
        error: _0x3af8c7[_0x3a9e5c(0x182)] || _0x2207c6[_0x3a9e5c(0x170)],
      })
    );
  }
}),
  (module[_0xe1f033(0x180)] = router);
