// live-stream-server/service.js
function _0x1792() {
  const _0x17836b = [
    "./auth",
    "odel",
    "800mhsbVb",
    "No\x20data\x20fo",
    "auth",
    "Router",
    "log",
    "16oSkxdd",
    "1419BZZDWz",
    "../../serv",
    "exports",
    "status",
    "5TGsNHs",
    "er/setting",
    "126VcktjR",
    "14643iBdypH",
    "524212mKtEGK",
    "get",
    "express",
    "/setting.m",
    "und!",
    "41052WTDlbF",
    "26438pDkLCk",
    "8nVxHbr",
    "Server\x20Err",
    "NfzUC",
    "TSLpJ",
    "findOne",
    "message",
    "1539570XtNiEw",
    "781vBNMTN",
    "1073772nFrpDW",
    "json",
    "Success!!",
    "1680hWFBRU",
    "NShHR",
  ];
  _0x1792 = function () {
    return _0x17836b;
  };
  return _0x1792();
}
const _0x4dfac8 = _0x3b2a;
function _0x3b2a(_0x3e2292, _0x5d7bc7) {
  const _0x568eba = _0x1792();
  return (
    (_0x3b2a = function (_0x15a599, _0x396881) {
      _0x15a599 = _0x15a599 - (0x88e + 0x77d * -0x5 + 0x1e35);
      let _0xf391a8 = _0x568eba[_0x15a599];
      return _0xf391a8;
    }),
    _0x3b2a(_0x3e2292, _0x5d7bc7)
  );
}
(function (_0x4fa727, _0x36f6bb) {
  const _0x1e9956 = _0x3b2a,
    _0x37dcb7 = _0x4fa727();
  while (!![]) {
    try {
      const _0x258d9d =
        (parseInt(_0x1e9956(0x16a)) / (0x1c9f + 0x27 * -0x86 + -0x834)) *
          (-parseInt(_0x1e9956(0x155)) / (0x239 + 0x846 + -0xa7d)) +
        (parseInt(_0x1e9956(0x16b)) / (-0x4af + 0x1d * -0x9e + -0xf1 * -0x18)) *
          (-parseInt(_0x1e9956(0x165)) /
            (0x328 * 0x6 + 0x2014 + 0x66 * -0x80)) +
        (parseInt(_0x1e9956(0x16f)) /
          (0x1 * 0x1b73 + 0x10 * -0x236 + 0x153 * 0x6)) *
          (-parseInt(_0x1e9956(0x15c)) / (0x1 * -0x1264 + 0x114 + 0x1156)) +
        (-parseInt(_0x1e9956(0x15e)) / (0xa13 + 0x39 * -0x2e + -0x2 * -0x19)) *
          (parseInt(_0x1e9956(0x156)) / (0x25e9 + -0x1 * -0x1f5 + -0x27d6)) +
        (parseInt(_0x1e9956(0x172)) /
          (-0x5cb * -0x1 + -0x2186 + 0x1 * 0x1bc4)) *
          (parseInt(_0x1e9956(0x161)) / (-0x8e5 + 0x427 * -0x1 + 0x19 * 0x86)) +
        (-parseInt(_0x1e9956(0x15d)) /
          (-0x17 * 0x8d + 0x923 * -0x4 + 0x2 * 0x18a1)) *
          (-parseInt(_0x1e9956(0x154)) / (0x75 * -0xb + 0x10c9 + -0xbb6)) +
        (parseInt(_0x1e9956(0x173)) / (-0xc7f + 0x16ef + -0xa63)) *
          (parseInt(_0x1e9956(0x171)) / (-0xc6c + 0xe24 * -0x2 + 0x28c2));
      if (_0x258d9d === _0x36f6bb) break;
      else _0x37dcb7["push"](_0x37dcb7["shift"]());
    } catch (_0x4367f5) {
      _0x37dcb7["push"](_0x37dcb7["shift"]());
    }
  }
})(_0x1792, -0x1907d + -0x18ee * -0x19 + 0x19e27);
const express = require(_0x4dfac8(0x175)),
  router = express[_0x4dfac8(0x168)](),
  data = require(_0x4dfac8(0x163)),
  Setting = require(_0x4dfac8(0x16c) +
    _0x4dfac8(0x170) +
    _0x4dfac8(0x152) +
    _0x4dfac8(0x164));
router[_0x4dfac8(0x174)]("/", async (_0x23a39e, _0x17c083) => {
  const _0x3e53d1 = _0x4dfac8,
    _0x4816a1 = {
      NShHR: _0x3e53d1(0x166) + _0x3e53d1(0x153),
      NfzUC: _0x3e53d1(0x160),
      TSLpJ: _0x3e53d1(0x157) + "or",
    };
  try {
    const _0x548618 = await Setting[_0x3e53d1(0x15a)]({});
    if (!_0x548618 || !data[_0x3e53d1(0x167)])
      return _0x17c083[_0x3e53d1(0x16e)](
        -0x315 * 0xc + -0x1 * 0x3f3 + 0xb5 * 0x3b
      )[_0x3e53d1(0x15f)]({
        status: ![],
        message: _0x4816a1[_0x3e53d1(0x162)],
      });
    return _0x17c083[_0x3e53d1(0x16e)](
      -0x6be * 0x1 + 0x12b9 * -0x1 + -0x1 * -0x1a3f
    )[_0x3e53d1(0x15f)]({
      status: !![],
      message: _0x4816a1[_0x3e53d1(0x158)],
      setting: _0x548618,
    });
  } catch (_0x56f43e) {
    return (
      console[_0x3e53d1(0x169)](_0x56f43e),
      _0x17c083[_0x3e53d1(0x16e)](-0x1485 + -0x1b58 + -0x109b * -0x3)[
        _0x3e53d1(0x15f)
      ]({
        status: ![],
        error: _0x56f43e[_0x3e53d1(0x15b)] || _0x4816a1[_0x3e53d1(0x159)],
      })
    );
  }
}),
  (module[_0x4dfac8(0x16d)] = router);
